### DISCUSSION TITLE
Idea: Press any key

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#141] [Ideas]
I would say that it would be fine that when you put `keyIsPressed()` (or any key event) without any parameter, that it returns `true` when any key is pressed. Alternative: `keyIsPressed("any")`

**Comment by slmjkdbtl** [comment]
I've been meaning to do that 👍 also `keyPress(() => {})` without param

**Comment by lajbel** [comment]
Cool! Was added to proect board https://github.com/replit/kaboom/projects/1

**Reply by lajbel** [reply]
Added in kaboom@next
